threadfin/examples/generic_samples/setupsample.yaml

```diff
@@ -0,0 +1,20 @@
+
+template_dir: "XX_PATH_TO_OTS-BOOKKEEPING_XX/threadfin/reconcile/templates"
```

Comment:
Why did you switch to hyphen instead of underscore in the middle of the variable?

Reply:
Because "OTS-BOOKKEEPPING" is the name of the repository. It is not two terms, rather one term specifically with that character connecting them. I thought it would be could to be consistent with the name

Reply:
Ah, gotcha.  However, the repository is also lower case... meanwhile, there's an environment variable whose name is "OTS_BOOKKEEPING", which is probably where the user is going to look anyway to figure out where their ${OTS_BOOKEEPING} dir is.  I would just use underscore here; it corresponds to the env variable, and it won't confuse the user.

Your response made sense, btw!  It's a judgement call.  It's also not the most important thing in the world, of course.  But IMHO just using underscores consistently here will be slightly easier for the user.  You can also put a comment above that line explaining what "OTS_BOOKKEEPING" refers to, if YAML supports comments (I don't know if it does).